torch/csrc/jit/codegen/cuda/lower_thread_predicate.cpp

```diff
@@ -392,7 +330,7 @@ ParallelTypeBitmap ThreadPredicateMap::getParallelBroadcastDomains(
     if (!id->isBroadcast()) {
       continue;
     }
-    if (id->isBlockDim() || (!output_smem && id->isThreadDim())) {
+    if (id->isThread() || (!output_smem && id->isThreadDim())) {
```

Comment:
Why is this changed? If a tensor is on shared memory, it doesn't need any specific broadcast operation other than just storing and syncthreads.

Reply:
Thank you, good catch.